driver-service/src/test/java/com/cabaggregator/driverservice/integration/controller/api/CarControllerIntegrationTest.java

```diff
@@ -83,6 +92,7 @@ void getPageOfCars_ShouldReturnCars_WhenTheyExist() {
         int expectedContentSize = 3;
 
         mockMvc.perform(get(baseUrl)
+                        .header("Authorization", authTestUtil.getAdminBearerToken())
```

Comment:
Здесь и далее можно воспользоваться имеющейся константой из `org.springframework.http.HttpHeaders`:
```suggestion
                        .header(HttpHeaders.AUTHORIZATION, authTestUtil.getAdminBearerToken())
```